src/components/generics/vacation-test-forms/employeeVacationRequests.tsx

```diff
@@ -233,26 +172,41 @@ const renderEmployeeVacationRequests = () => {
         marginBottom: 4
       }}
     >
-      <InputLabel>{ strings.editorContent.vacationType }</InputLabel>
+      <InputLabel>{ strings.vacationRequests.vacationType }</InputLabel>
       <Select
         value={ vacationType }
         onChange={ handleVacationTypeChange }
-        label={ strings.editorContent.vacationType }
+        label={ strings.vacationRequests.vacationType }
       >
-        <MenuItem value="Paid leave">{ strings.editorContent.paidLeave }</MenuItem>
-        <MenuItem value="Maternity leave">{ strings.editorContent.maternityLeave }</MenuItem>
-        <MenuItem value="Parental leave">{ strings.editorContent.parentalLeave }</MenuItem>
-        <MenuItem value="Unpaid leave">{ strings.editorContent.unpaidLeave }</MenuItem>
-        <MenuItem value="Surplus balance">{ strings.editorContent.surplusBalance }</MenuItem>
+        <MenuItem value={ VacationType.VACATION }>
+          { strings.vacationRequests.vacation }
+        </MenuItem>
+        <MenuItem value={ VacationType.UNPAID_TIME_OFF}>
+          { strings.vacationRequests.unpaidTimeOff }
+        </MenuItem>
+        <MenuItem value={ VacationType.SICKNESS}>
+          { strings.vacationRequests.sickness }
+        </MenuItem>
+        <MenuItem value={ VacationType.PERSONAL_DAYS }>
+          { strings.vacationRequests.personalDays }
+        </MenuItem>
+        <MenuItem value={ VacationType.MATERNITY_PATERNITY }>
+          { strings.vacationRequests.maternityPaternityLeave }
+        </MenuItem>
+        <MenuItem value={ VacationType.CHILD_SICKNESS }>
+          {strings.vacationRequests.childSickness }
+        </MenuItem>
       </Select>
     </FormControl>
   );
 
   /**
-   * Handle employee change
+   * Handle status change
+   * 
+   * @param event Select change event
    */
   const handleStatusChange = (event: SelectChangeEvent) => {
-    const contentValue = event.target.value;
+    const contentValue = event.target.value as VacationRequestStatus;
```

Comment:
Now that you implemented the type checking solution for handleFilterScopes you might as well do it here as well.